likelihood/2pt/spec_tools.py

```diff
@@ -94,9 +94,29 @@ def __init__(self, name, types, nbin_a, nbin_b, angles, spectra, is_auto,
             if bin_pair not in spectra:
                 raise ValueError("Missing spectrum pairs {}".format(bin_pair))
 
+        #Useful when we have multiple two point functions that we want to label separately
```

Comment:
I don't really undertand this - could you add comments explaining how to use this?  Also good to remove all the commented out stuff.